source/common/grpc/common.cc

```diff
@@ -24,6 +24,20 @@ namespace Grpc {
 
 const std::string Common::GRPC_CONTENT_TYPE{"application/grpc"};
 
+Status::GrpcStatus Common::getGrpcStatus(const Http::HeaderMap& trailers) {
```

Comment:
Sure.  It's a minor behavior change for out-of-bounds error codes.  If we care we could split returns from getGrpcStatus into "missing" status and "out of bounds" status but either way we'll throw an exception so I assume it's moot.